src/com/aca/homework/week4/snake/Board.java

```diff
@@ -0,0 +1,105 @@
+package com.aca.homework.week4.snake;
+
+
+import java.util.Random;
+import java.util.Scanner;
+
+public class Board {
+
+    public static void main(String[] args) {
+        Snake[][] board = new Snake[8][8];
+
+        for (int i = 0; i < 8; i++) {
+            for (int j = 0; j < 8; j++) {
+                board[i][j] = new Snake(i, j);
+            }
+        }
+
+        board[0][0].setSnakeHead();
```

Comment:
I suggest moving these 2 lines to a new method having a name initateHeadAndApple